query/src/main/scala/filodb/query/exec/ExecPlan.scala

```diff
@@ -249,6 +273,40 @@ abstract class LeafExecPlan extends ExecPlan {
   final def children: Seq[ExecPlan] = Nil
 }
 
+sealed trait FuncArgs{
+  def getResult (implicit sched: Scheduler, timeout: FiniteDuration) : Observable[ScalarVector]
+}
+case class ExecPlanFuncArgs(execPlan: ExecPlan) extends FuncArgs {
```

Comment:
final case class, to make sure it's not extendable.  All case classes need to have final

Reply:
This is also not the file I checked in yesterday. I have made it final.  Git UI is showing me```Outdated```